eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -0,0 +1,220 @@
+$GithubAPIBaseURI = "https://api.github.com/repos"
+
+function Get-Headers ($token) {
```

Comment:
I suggest calling this `Get-GithubHeaders` since it is specific to our github usage.